elastic/kibana
# [Fleet] added missing cd command to Windows instructions

## Summary

Fixes https://github.com/elastic/kibana/issues/128342

Steps to verify:
- Open Add agent flyout (with enrolled Fleet Server)
- Select a policy and choose Windows platform to see instructions
- Verify that cd command is there

<img width="845" alt="image" src="https://user-images.githubusercontent.com/90178898/161052970-dde3cee5-23de-43f2-86eb-906b530f543f.png">
